x/rollapp/genesis.go

```diff
@@ -11,24 +12,43 @@ import (
 func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
 	// Set all the rollapp
 	for _, elem := range genState.RollappList {
+
+		// validate rollapp info
+		err := elem.ValidateBasic()
+		if err != nil {
+			k.Logger(ctx).Error("error init genesis validating rollapp information: rollapp:%s", elem.RollappId)
+			continue
```

Comment:
genesis should fail when bad genesis provided

Reply:
done